.github/workflows/test.yml

```diff
@@ -4,41 +4,20 @@ name: test
 on:
   push:
     branches:
-    - master
+    - main
     tags:
     - v*
   pull_request:
 env:
   DEBIAN_FRONTEND: noninteractive
 jobs:
   lint:
-    strategy:
-      matrix:
-        os: [ubuntu-latest]
-    runs-on: ${{ matrix.os }}
-    steps:
-      - uses: actions/checkout@v3
-      - uses: golangci/golangci-lint-action@v3
-        with:
-          args: --timeout 2m
+    uses: mackerelio/workflows/.github/workflows/go-lint.yml@v1.1.0
   test:
-    strategy:
-      matrix:
-        go: ["1.19.x", "1.18.x"]
-        os: [ubuntu-latest]
-    runs-on: ${{ matrix.os }}
-    steps:
-    - uses: actions/setup-go@v2
-      with:
-        go-version: ${{ matrix.go }}
-    - uses: actions/checkout@v2
-    - uses: actions/cache@v2
-      with:
-        path: ~/go/pkg/mod
-        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
-        restore-keys: |
-          ${{ runner.os }}-go-
-    - run: |
-        make test
-    - run: |
-        ./test.sh
+    uses: mackerelio/workflows/.github/workflows/go-test.yml@v1.1.0
+  testci:
+    uses: mackerelio/workflows/.github/workflows/setup-go-matrix.yml@v1.1.0
+    with:
+      os-versions: '["ubuntu-22.04"]'
+      run: |
+        make testci
```

Comment:
i think need a build process.

Reply:
Actually, **test.sh** will implicit build this plugin:

https://github.com/mackerelio/mackerel-plugin-mysql/blob/ba7afc58d8bbb6471054e3cc573d6f516762ed1d/test.sh#L11